linux/drivers/hid/hid-magicmouse.c

```diff
@@ -928,8 +933,13 @@ static void magicmouse_remove(struct hid_device *hdev)
 	hid_hw_stop(hdev);
 }
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 12, 0)
```

Comment:
> This function isn't used outside of this file so I can't figure out why this compatibility is needed. I wonder if this is related to the error that @jslay88 is seeing?

https://lore.kernel.org/all/20240803-hid-const-fixup-v2-6-f53d7a7b29d8@weissschuh.net/

Because of this change, the kernel version needs to be checked during compilation.

Reply:
I see, thanks.